source/scripting_v3/GTA/Game.cs

```diff
@@ -78,6 +78,16 @@ public static Player Player
 			}
 		}
 
+		/// <summary>
+		/// Gets the main blip of the <see cref="GTA.Player"/> that you are controlling.
+		/// </summary>
+		public static Blip MainPlayerBlip => new Blip(Function.Call<int>(Hash.GET_MAIN_PLAYER_BLIP_ID));
```

Comment:
Probably should be named `PlayerBlip`, for consistency with the `Game.Player` property.